model/src/pyrenew/deterministic/deterministic.py

```diff
@@ -17,26 +17,26 @@ class DeterministicVariable(RandomVariable):
 
     def __init__(
         self,
-        vars: ArrayLike,
         name: str,
+        vars: ArrayLike,
     ) -> None:
         """Default constructor
 
         Parameters
         ----------
-        vars : ArrayLike
-            A tuple with arraylike objects.
         name : str
             A name to assign to the process.
+        vars : ArrayLike
+            A tuple with arraylike objects.
```

Comment:
Good opportunity to fix this.
```suggestion
            An ArrayLike object.
```